common-rest/src/test/java/org/uniprot/api/common/repository/search/facet/FacetResponseConverterTest.java

```diff
@@ -79,6 +79,36 @@ void convertFacetWithoutLabel() {
         assertEquals(Long.valueOf(11L), itemValue.getCount());
     }
 
+    @Test
+    void convertFacetWithoutLabelDescendingValueSort() {
+        List<FacetField> fieldList = getFacetFields("annotation");
+
+        when(queryResponse.getFacetFields()).thenReturn(fieldList);
+
+        FacetResponseConverter facetConverter = new FacetResponseConverter(new FakeFacetConfig());
+        List<Facet> facets = facetConverter.convert(queryResponse);
+        assertNotNull(facets);
+
+        Facet annotation = facets.get(0);
+        assertEquals("Annotation", annotation.getLabel());
+        assertEquals("annotation", annotation.getName());
+        assertTrue(annotation.isAllowMultipleSelection());
+        assertNotNull(annotation.getValues());
+        assertEquals(5, annotation.getValues().size());
+
+        FacetItem itemValue = annotation.getValues().get(0);
+        assertNotNull(itemValue);
+        assertNull(itemValue.getLabel());
+        assertEquals("5", itemValue.getValue());
+        assertEquals(Long.valueOf(51L), itemValue.getCount());
+
+        itemValue = annotation.getValues().get(1);
+        assertNotNull(itemValue);
+        assertNull(itemValue.getLabel());
+        assertEquals("4", itemValue.getValue());
+        assertEquals(Long.valueOf(41L), itemValue.getCount());
+    }
```

Comment:
is it possible for there to be two values with the same count? Is it easy to add another section like lines 105-109, with itemValue.getValue() == 4 again, just to nail down that edge case?

Reply:
It is not possible because we populate it in a fake mocked config.